hail/src/main/scala/is/hail/expr/ir/Pretty.scala

```diff
@@ -117,6 +117,13 @@ object Pretty {
 
       sb.append(prettyClass(ir) )
 
+      /**
```

Comment:
delete?

Reply:
left this intentionally, it's quite useful for debugging.